statistics/handle/update.go

```diff
@@ -484,6 +484,15 @@ func (h *Handle) DumpStatsDeltaToKV(mode dumpMode) error {
 
 // dumpTableStatDeltaToKV dumps a single delta with some table to KV and updates the version.
 func (h *Handle) dumpTableStatCountToKV(id int64, delta variable.TableDelta) (updated bool, err error) {
+	statsVer := uint64(0)
+	defer func() {
+		if err != nil {
+			return
+		}
+		if statsVer != 0 {
```

Comment:
How about using `if err == nil && statsVer != 0` to make it more concise?

Reply:
Done